libuuu/fastboot.cpp

```diff
@@ -354,7 +354,7 @@ int FBCopy::run(CmdCtx *ctx)
 			if (sz > m_Maxsize_pre_cmd)
 				sz = m_Maxsize_pre_cmd;
 
-			cmd.format("donwload:%08X", sz);
+			cmd.format("download:%08X", sz);
```

Comment:
Even it spell error, but still can't direct change it now. 

Need update imx-uuc firstly.  

https://github.com/NXPmicro/imx-uuc/blob/master/ufb.c

} else if(strncmp(cmd, "donwload:", 9) == 0) {

Add need keep compatibility for time.